windows-container-samples/ASP-NET-Blog-Application/db/dockerfile

```diff
@@ -0,0 +1,22 @@
+
+# SQL Express Base Image 
+# NOTE: You may need to prepend 'microsoft/' to base image name--use 'docker images' to see what your image is called.
+FROM microsoft/mssql-server-2014-express-windows
+
+# SQL environmental variables (replace "Password123" with your sa password)
+ENV sa_password "Password123"
```

Comment:
@friism  - Thanks for the feedback. We'll look into using ADD <url> for the Blog Engine files (http://dotnetblogengine.net/) and then just copy the web.config file which was customized